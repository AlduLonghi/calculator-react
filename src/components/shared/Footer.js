import '../../style/App.css';
import logo from '../../assets/m.svg';

const Footer = () => (
  <footer className="d-flex">
    <img src={logo} alt="logo" />
  </footer>
);

export default Footer;
