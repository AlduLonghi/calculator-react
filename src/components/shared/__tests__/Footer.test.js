import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

it('Snapshot test for Footer component', () => {
    const tree = renderer
    .create(<Footer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})