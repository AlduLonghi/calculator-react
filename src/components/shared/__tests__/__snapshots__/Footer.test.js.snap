// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Footer component 1`] = `
<footer
  className="d-flex"
>
  <img
    alt="logo"
    src="m.svg"
  />
</footer>
`;
